mm0-rs/src/server.rs

```diff
@@ -1195,13 +1222,25 @@ impl Server {
             if let Some((id, req)) = parse_request(req)? {
               spawn_new(|cancel| {
                 reqs.ulock().insert(id.clone(), cancel.clone());
-                async {
-                  RequestHandler {id, cancel}.handle(req).await
+                let elab_on = options.ulock().elab_on.unwrap_or_default();
```

Comment:
Instead of making this an extra parameter in `handle` et al, how about adding a `elab_on()` method that gets this from `SERVER` and use it in the requests that need it?